pdns/recursordist/docs/lua-scripting/hooks.rst

```diff
@@ -24,8 +24,12 @@ It also means that to print such names, the ``:toString`` method must be used (o
 Once a script is loaded, PowerDNS looks for the interception functions in the loaded script.
 All of these functions are optional.
 
-If a function returns true, it will indicate that it handled a query.
-If it returns false, the Recursor will continue processing unchanged (with one minor exception).
+If ``ipfilter`` returns ``true``, the query is dropped.
+If ``preresolve`` returns ``true``, it will indicate it handled a query, and the recursor will send to result as constructed in the functions to the client.
```

Comment:
```suggestion
If ``preresolve`` returns ``true``, it will indicate it handled a query, and the recursor will send the result as constructed in the function to the client.
```